distributed/tests/test_worker.py

```diff
@@ -2011,18 +2013,18 @@ def get_thread_name():
             s.address,
             nthreads=2,
             executor={
-                "GPU": ThreadPoolExecutor(1, thread_name_prefix="Dask-GPU-Threads")
+                "foo": ThreadPoolExecutor(1, thread_name_prefix="Dask-Foo-Threads")
             },
         ) as w:
             async with Client(s.address, asynchronous=True) as c:
                 futures = []
                 with dask.annotate(executor="default"):
                     futures.append(c.submit(get_thread_name, pure=False))
-                with dask.annotate(executor="GPU"):
+                with dask.annotate(executor="foo"):
                     futures.append(c.submit(get_thread_name, pure=False))
-                default_result, gpu_result = await c.gather(futures)
+                default_result, foo_result = await c.gather(futures)
                 assert "Dask-Default-Threads" in default_result
-                assert "Dask-GPU-Threads" in gpu_result
+                assert "Dask-Foo-Threads" in foo_result
```

Comment:
Note that this test was totally fine before, I'm just changing the name of the extra executor from "GPU" to "foo" to avoid any potential confusion with the "gpu" executor which is added in this PR